

--- cmisc.c 2005-05-13 17:52:24.584869184 -0400
+++ cmisc-new.c 2005-05-13 17:53:43.114930800 -0400
@@ -291,7 +291,13 @@
descUpdateFunc = update;
describeW_posy = 0;
if ( describePG.win == NULL ) {
- ParamCreateDialog( &describePG, "Description", "Done", DescOk, NULL, TRUE,
DescribeLayout, F_RECALLPOS, DescribeUpdate );
+ /* SDB 5.13.2005 */
+ ParamCreateDialog( &describePG, "Description", "Done", DescOk,
+ (paramActionCancelProc) DescribeCancel,
+ TRUE, DescribeLayout, F_RECALLPOS,
+ DescribeUpdate );
+
+ /* ParamCreateDialog( &describePG, "Description", "Done", DescOk, NULL,
TRUE, DescribeLayout, F_RECALLPOS, DescribeUpdate ); */
describeCmdButtonEnd = wControlBelow( (wControl_p)describePG.helpB );
}
for ( inx=0; inx<sizeof describePLs/sizeof describePLs[0]; inx++ ) {